internal/store/fleet.go

```diff
@@ -155,6 +155,7 @@ func (s *FleetStore) createFleet(fleet *model.Fleet) (bool, error) {
 	if fleet.Spec.Data.Template.Metadata == nil {
 		fleet.Spec.Data.Template.Metadata = &api.ObjectMeta{}
 	}
+
 	fleet.Spec.Data.Template.Metadata.Generation = lo.ToPtr[int64](1)
```

Comment:
```suggestion
	fleet.Spec.Data.Template.Metadata.Generation = util.IntToPtr(1)
```

Reply:
Removed the empty line - not part of this PR